src/libraries/Microsoft.NETCore.Platforms/src/RuntimeGroupCollection.cs

```diff
@@ -34,7 +34,7 @@ public RuntimeGroupCollection(ICollection<RuntimeGroup> runtimeGroups)
         /// <param name="parent"></param>
         public void AddRuntimeIdentifier(string runtimeIdentifier, string parent)
         {
-            RID rid = RID.Parse(runtimeIdentifier);
+            RID rid = RID.Parse(runtimeIdentifier, noQualifier: true);
```

Comment:
Nit: it would be good to add a comment.
```suggestion
            // don't parse qualifier since we don't use them and they are ambiguous with `-` in base RID
            RID rid = RID.Parse(runtimeIdentifier, noQualifier: true);
```